tests/test_cases/test_failure/Makefile

```diff
@@ -0,0 +1,14 @@
+# Copyright cocotb contributors
+# Licensed under the Revised BSD License, see LICENSE for details.
+# SPDX-License-Identifier: BSD-3-Clause
+
+MODULE := test_failure
+
+# squash error code from simulator and ensure the cocotb test passed
```

Comment:
This comment is a bit confusing, given that we're testing the xunit reporting does something sensible when CI does _not_ pass.